crates/sui-rpc-benchmark/src/json_rpc/runner.rs

```diff
@@ -305,8 +317,8 @@ pub async fn run_queries(
                                     }
                                     is_error = false;
                                 } else {
-                                    warn!("JSON parsing error for method: {}, error: {:?}, response: {}", 
-                                        request_line.method, parse_result.err(), resp_text);
+                                    warn!("Response received but JSON parsing failed for method: {}, body: {:?}, error: {:?}, response: {}", 
+                                        request_line.method, request_line.body_json, parse_result.err(), resp_text);
```

Comment:
Use trace parameters.

```suggestion
                                    warn!(
                                        method = request_line.method,
                                        body = ?request_line.body_json,
                                        error = ?parse_result.err(),
                                        response = resp_text,
                                        "Response received but JSON parsing failed"
                                    );
```